src/main/java/com/org/ita/kata/implementation/MyroslavaVynokur/EigthImpl.java

```
@@ -14,8 +14,15 @@ public double getVolumeOfCuboid(double length, double width, double height) {
     }
 
     @Override
-    public float mpgToKPM(float mpg) {
-        return 0;
+    public float mpgToKPM(float mpg) throws ArithmeticException {
```

need to throw the error above?

---

removed error